src/runtime/contrib/tensorrt/tensorrt_module.cc

```diff
@@ -59,6 +63,10 @@ class TensorRTModule : public runtime::ModuleNode {
     for (auto& it : trt_engine_cache_) {
       it.second.context->destroy();
       it.second.engine->destroy();
+      for (void* ptr : it.second.device_mem_pointers) {
+        if (ptr)
```

Comment:
I think according to google style guide this type of if is only allowed if on same line like:
`if (ptr) TVMDeviceFreeDataSpace({kDLGPU, 0}, ptr);`

Otherwise, we need to add `{ }`.

Reply:
thanks, I will make that change.